elfloader-tool/src/arch-riscv/console.c

```diff
@@ -5,7 +5,7 @@
  */
 
 #include <elfloader_common.h>
-#include "sbi.h"
+#include <sbi.h>
```

Comment:
This should be in another commit?

Reply:
Or this change can be dropped as it doesn't affect the rest of the PR I believe.

Reply:
Yeah thought I would just fix it up. Not sure if it was intended.